Ginger/GingerCoreNET/RunLib/CLILib/CLIDynamicFile.cs

```diff
@@ -249,6 +257,44 @@ private void LoadSealightsDetailsFromJSON(GingerExecConfig gingerExecConfig)
             WorkSpace.Instance.Solution.SealightsConfiguration.SealightsReportedEntityLevel = (SealightsConfiguration.eSealightsEntityLevel)gingerExecConfig.SealightsDetails.SealightsEntityLevel;
             WorkSpace.Instance.Solution.SealightsConfiguration.SealightsAgentToken = gingerExecConfig.SealightsDetails.SealightsAgentToken;
         }
+        private void LoadGlobalVariableValues(GingerExecConfig gingerExecConfig)
+        {
+            try
+            {
+                foreach (GlobalVariable customizedVar in gingerExecConfig.GlobalVariables)
+                {
+                    VariableBase originalVar = DynamicExecutionManager.FindItemByIDAndName<VariableBase>(
+                                   new Tuple<string, Guid?>(nameof(VariableBase.Guid), customizedVar.VariableID),
+                                   new Tuple<string, string>(nameof(VariableBase.Name), customizedVar.VariableName),
+                                   WorkSpace.Instance.Solution.Variables);
+                    if (originalVar != null)
+                    {
+                        switch (originalVar.VariableType)
+                        {
+                            case "String":
+                            case "Number":
+                            case "Selection List":
+                            case "DateTime":
+                                if (!originalVar.SetValue(customizedVar.VariableCustomizedValue))
+                                {
+                                    throw new InvalidOperationException($"Invalid value/format '{customizedVar.VariableCustomizedValue}' provided for variable '{originalVar.Name}'");
+                                }
+                                break;
+                            default: throw new InvalidOperationException($"Value customization is not allowed for variable '{originalVar.Name}'. Only String, Number,Selection List and DateTime variable value customization is allowed");
```

Comment:
Change message as "Value customisation is not supported for variable type {Type of variable}, variable name {Name}

Reply:
done